src/documents/management/commands/manage_superuser.py

```diff
@@ -12,31 +12,42 @@ class Command(BaseCommand):
 
     help = """
         Creates a Django superuser based on env variables.
+        PAPERLESS_ADMIN_USER name of admin user (Default: admin)
+        PAPERLESS_ADMIN_MAIL admin email address (Default: root@localhost)
+        PAPERLESS_ADMIN_PASSWORD (NODEFAULT)
+
+        Logic:
+              Check if password is set, if not exit
+              Check is admin user exists, if exists exit
+                                          else create
     """.replace("    ", "")
 
     def handle(self, *args, **options):
+        # Get info from env
+        username = os.getenv('PAPERLESS_ADMIN_USER', 'admin')
+        mail = os.getenv('PAPERLESS_ADMIN_MAIL', 'root@localhost')
+        password = os.getenv('PAPERLESS_ADMIN_PASSWORD')
 
-        username = os.getenv('PAPERLESS_ADMIN_USER')
-        if not username:
+        # Return if email address does not pass basic validation
+        if not re.fullmatch(r"[^@]+@[^@]+", mail):
```

Comment:
Django contains an EmailValidator. The root@localhost default could be added to the allowlist. I think using that validator would be cleaner.
https://docs.djangoproject.com/en/4.0/ref/validators/#emailvalidator

Reply:
Good point, I'll adopt it.
I added a minimal validation just to show that it might be a good idea. Did not thought about django giving what I need.